src/angular/header-lean/app-chooser-section.scss

```diff
@@ -47,4 +47,20 @@
   &:last-child {
     border-bottom: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
   }
+
+  > :is(.sbb-icon, img) {
```

Comment:
I'm not sure whether we should use '>' or not. I think an icon inside an icon will never happen, but maybe someone uses a span inside the `<a>` or whatever. What do you think?

Reply:
Makes sense if we want to allow child elements